app/assets/javascripts/script.coffee

```diff
@@ -0,0 +1,11 @@
+$ ->
```

Comment:
you can even wrap that all in class. That's one of main advantages of Coffeescript

it'll be +- something like:
```
class OrderForm
  FOOD_ITEM_SELECTOR = '.js-food-item'

  constructor: (form) ->
    @form = $(form)

  init: ->
    @bindEvents()

  bindEvents: ->
    @form.on 'change', @FOOD_ITEM_SELECTOR, ->
      let price = $("#{@FOOD_ITEM_SELECTOR}:selected").data('price')

      this.attr('min', price).val(price)

$ ->
  new OrderForm('#js_main').init()

```
[source](https://coffeescript.org)